### ISSUE TITLE
Default Chart name

### ISSUE THREAD

**Issue by james-rae** [ramp4-pcar4/highcharts-editor#55]
## Description

When in French from the very start of a session, the chart generates with the title `Basic Chart`. Should be lang-specific default
